tests/test_cli.py

```diff
@@ -212,6 +212,23 @@ def test_command_pipeline(self):
         with io.open(self.image_5050_b64) as image_b64:
             assert result.output == image_b64.read()
 
+    def test_command_pipeline_all_actions(self):
+        actions = json.dumps([
+            {"action": "denoise", "options": {"value": 9}},
+            {"action": "equalize", "options": {"value": 10}},
+            {"action": "brightness", "options": {"value": 122}},
+            {"action": "contrast", "options": {"value": 122}},
+            {"action": "smooth", "options": {"value": 12}},
+            {"action": "posterize", "options":
+                {"colors": 4, "method": "kmeans"}}
```

Comment:
That actually should have been `linear`. With `kmeans` we cannot guarantee the test is reproducible.